### DISCUSSION TITLE
Cookie-based authentication

### DISCUSSION THREAD

**Discussion by savornicesei** [warmuuh/milkman#159] [Q&A] [answered]
Hi all,

Does Milkman Scripting plugin work with cookie-based authentication where the auth, cookie is a http-only cookie?

Thank you.

**Comment by warmuuh** [comment, marked as answer]
as cookies are simply headers, it works similar. you need to search for the right cookie in milman.response.header.entries (array of {key:.., value:..} objects). after your login-call, you have to look for the [Set-Cookie](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie) header and parse the value.

when sending an authenticated request, you can set the Cookie header accordingly then

there is no extra support for cookies though (yet? ;))
